<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DicomViewer</class>
 <widget class="QWidget" name="DicomViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dicom Viewer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="CustomStyle" stdset="0">
        <string>dcmviewer_tool_bg</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="fileBtn">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;检索并加载文件夹图像&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/png/open.png</normaloff>:/png/open.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+F</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="exportBtn">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;导出图像&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/png/export.png</normaloff>:/png/export.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+E</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="gridBtn">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;视图分格&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/png/grid.png</normaloff>:/png/grid.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+G</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="annoBtn">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;显示/隐藏角注&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/png/anno.png</normaloff>:/png/anno.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+A</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="adjustBtn">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;调整窗宽窗位&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/png/adjust.png</normaloff>:/png/adjust.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+D</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="panBtn">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;图像平移&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/png/move.png</normaloff>:/png/move.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+M</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="zoomBtn">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;图像缩放&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/png/zoomin.png</normaloff>:/png/zoomin.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="slicingBtn">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;浏览序列&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/png/slicing.png</normaloff>:/png/slicing.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::DelayedPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="marksBtn">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;标注和测量&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/png/line.png</normaloff>:/png/line.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="flipBtn">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>38</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;图像翻转&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/png/hflip.png</normaloff>:/png/hflip.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>false</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
          <property name="CustomStyle" stdset="0">
           <string>comb_dcmviewer</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="fullScreenBtn">
          <property name="minimumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/png/full_screen.png</normaloff>:/png/full_screen.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="ViewContainerWidget" name="viewContainer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="ThumbnailBarWidget" name="thumbnailBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>475</width>
        <height>612</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewContainerWidget</class>
   <extends>QWidget</extends>
   <header location="global">view/KissView</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ThumbnailBarWidget</class>
   <extends>QWidget</extends>
   <header location="global">view/KissView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
